tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -189,20 +189,57 @@ describe('main_html5 wrapper tests', function () {
 
   it('should notify MULTI_AUDIO_AVAILABLE on first \'canPlay\' event', function(){
     wrapper.getAvailableAudio = function() {
-      return [{"id": 1}, {"id": 2}];
+      return [{
+        'id': 1,
+        'label': 'eng',
+        'lang': 'eng',
+        'enabled': true
+      }, {
+        'id': 2,
+        'label': 'ger',
+        'lang': 'ger',
+        enabled: false
+      }];
     };
-    $(element).triggerHandler("canplay");
+    $(element).triggerHandler('canplay');
     expect(vtc.notifyParameters[0]).to.eql(vtc.interface.EVENTS.MULTI_AUDIO_AVAILABLE);
   });
 
   it('should not notify MULTI_AUDIO_AVAILABLE on first \'canPlay\' event when getAvailableAudio returns too short array', function(){
     wrapper.getAvailableAudio = function() {
-      return [{"id": 1}];
+      return [{'id': 1}];
     };
-    $(element).triggerHandler("canplay");
+    $(element).triggerHandler('canplay');
     expect(vtc.notifyParameters[0]).to.not.eql(vtc.interface.EVENTS.MULTI_AUDIO_AVAILABLE);
   });
 
+  it('should notify MULTI_AUDIO_CHANGED after setAudio', function() {
+    element.audioTracks[0] = { id: 0, language: 'en', label: '', enabled: true };
+    element.audioTracks[1] = { id: 1, language: 'en', label: '', enabled: false };
+    
+    element.audioTracks.__proto__ = {
+      getTrackById: vtc.getTrackById
+    };
+
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
+    wrapper.setAudio(1);
```

Comment:
Are these many setAudio calls testing that multiple setAudio calls to the same audio id only triggers one callback?

Reply:
Nope, this is fine, was just curious. Thanks!